docs/docs/llms/openai/notebooks/openai-embeddings-generation.ipynb

```diff
@@ -133,7 +133,7 @@
     "   - Adds a space after specific HTML tags in a BeautifulSoup object for better text readability.\n",
     "\n",
     "2. **extract_text_from_url**: \n",
-    "   - Extracts text from a specified webpage section using its URL and a target ID. Filters and organizes the text from tags like <h>, <li>, and <p>, excluding certain irrelevant sections.\n",
```

Comment:
markdown cells are copied as-is to docusaurus, so we can't include these as they are invalid (no closing tag). actually, even in non-MDX context, this is a problem:

<img width="1020" alt="Screenshot 2024-11-28 at 11 27 16 AM" src="https://github.com/user-attachments/assets/8cf050e7-aea2-4dc1-b79c-7cfeaceeba77">